code/game/objects/items/tools/maintenance_tools.dm

```diff
@@ -557,13 +557,19 @@
 		if(requires_superstrength_pry)
 			if(!HAS_TRAIT(user, TRAIT_SUPER_STRONG)) //basically IS_PRY_CAPABLE_CROWBAR
 				return
-		if(!attacked_door.density) //If its open
-			return
 		if(attacked_door.heavy) //Unopenable
 			to_chat(usr, SPAN_DANGER("You cannot force [attacked_door] open."))
 			return
 		if(user.action_busy)
 			return
+		if(!attacked_door.density && !attacked_door.arePowerSystemsOn()) //If its open and unpowered
+			attacked_door.close(TRUE)
+			return
+		if(attacked_door.density && !attacked_door.arePowerSystemsOn()) // if its closed and unpowered
+			attacked_door.open(TRUE)
+			return
+		if(!attacked_door.density && attacked_door.arePowerSystemsOn()) //If its open and powered
```

Comment:
```suggestion
		if(!attacked_door.density) //If its open and powered
```
We already know the power systems are on because otherwise it would have entered the earlier check.

Reply:
I tried doing it without power check but it clocked me an error for some reason

Reply:
ok I tried it again and it worked for some reason

Reply:
The comment was still correct, but its fine. As I said we know it is powered.